src/app/components/homePage/shared/eventPortal.less

```diff
@@ -78,6 +78,7 @@
       justify-content: flex-start;
       border: solid;
       overflow: auto;
+      max-height: 300px;
```

Comment:
Can we remove this? Otherwise we restrict the size of jobs, and they can only see one at a time.